csfieldguide/templates/interactives/matrix-simplifier.html

```diff
@@ -106,6 +106,22 @@ <h5 class="col-12 text-center">{% trans "Result" %}</h5>
       </div>
     </div>
 
+    <div class="row justify-content-center mb-1">
+      <small class="text-muted text-center">{% trans "The format below can be copied for pasting into the scene editor interactive." %}</small>
+    </div>
+
+    <div class="row justify-content-center mb-2">
+      <textarea id="code-to-copy" class="form-control col-md-4 col-sm-12" readonly></textarea>
+    </div>
+
+    <div class="row justify-content-center mb-5">
+      <button id="copy-eqtn" type="button" class="btn btn-primary col-md-2 col-sm-6 d-none" data-toggle="tooltip" data-placement="bottom">{% trans "Copy equation" %}</button>
+    </div>
+
+    <div class="row justify-content-center">
+      <button id="remove-all" type="button" class="btn btn-danger btn-sm col-md-2 col-sm-6 d-none">{% trans "Remove all" %}</button>
+    </div>
+
```

Comment:
(nowhere in particular)

I'm not a fan of the layout here
![image](https://user-images.githubusercontent.com/25916475/66965443-17888000-f0d6-11e9-86ee-b230a4842054.png)
- What 'remove all' does is ambiguous, consider having two: one next to each 'add matrix/vector' button
- 'Copy equation' should specify that it'll be put on the clipboard i.e. 'copy to clipboard'
- Consider reducing the height of the code box to one line and using a scrollbar
- Consider: 'For convenience this code can be pasted into the Scene Editor interactive'